### PR TITLE
Utopia 307 ppq form

### PR DESCRIPTION
**Pull request by binggaofw**
# Description

This PR includes the following proposed change(s):
this is the first version of ppq form
--not add progress bar yet
--no unit test 
-- no lint  there is a @fortawesome/fontawesome-svg-core error i can not fix
Screen Shot 2022-10-18 at 8 02 11 AM

- {List all the changes, if possible add the jira ticket #}
ppq-form

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [ ] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ x] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [ x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

### CODE REVIEW

File: src/frontend/src/pages/PPQFormPage.tsx [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+              <div className="radio">
+                <label>
+                  <input type="radio" value="no" />
+                  No
+                </label>
+              </div>
+              <div className="radio">
+                <label>
+                  <input type="radio" value="notsure" />
+                  I'am not sure
```
[the remark this answers is not in the document]
Comment (binggaofw) [reply src/frontend/src/pages/PPQFormPage.tsx:173]: done

File: src/frontend/.dockerignore [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# Anything not used by the build system should go here.
+Dockerfile
+.dockerignore
+.gitignore
+README.md
+
+# Artifacts that will be built during image creation.
+# This should contain all files created during `npm run build`.
+build
+node_modules
```
Comment (Bottle7) [inline src/frontend/.dockerignore:11]: CRLF. Please leave an empty line at the end of files to avoid triggering @akroon3r

**Comment by Bottle7** [comment]
This doesn't build in docker for me. I have attached the error log.

[utopia-307-ppq-form_build-error.txt](https://github.com/bcgov/cirmo-dpia/files/9813731/utopia-307-ppq-form_build-error.txt)

**Comment by binggaofw** [comment]
> 

I will fix now

**Comment by binggaofw** [comment]
@Bottle7 can you try pull latest code and run in your local to see the result? thanks

**Comment by Bottle7** [comment]
> @Bottle7 can you try pull latest code and run in your local to see the result? thanks

Works!

**Review by Bottle7 — APPROVED** [review]
Looks good!

File: src/frontend/src/pages/PPQFormPage.tsx [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+                onClick={handleBackClick}
+              >
+                Back
+              </button>
+              <button className="btn-primary btn-next">Submit</button>
+            </div>
+          </form>
+        </div>
+      </section>
+      <Footer />
```
Comment (kushal-arora-fw) [inline src/frontend/src/pages/PPQFormPage.tsx:271]: @binggaofw - we should not need to inject Headers and Footers in every page. 

Approving for now

File: src/frontend/src/pages/PPQFormPage.tsx [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+                  </span>
+                  <div>
+                    <label>
+                      <input type="radio" value="yes" checked={true} />
+                      Yes
+                    </label>
+                  </div>
+                  <div className="radio">
+                    <label>
+                      <input type="radio" value="no" />
```
Comment (kushal-arora-fw) [inline src/frontend/src/pages/PPQFormPage.tsx:241]: Please refer to the enum values for them. May be work on them as you are integrating the API `POST /api/ppq`. Happy if you update it as part of the integration ticket/PR. This applies to all values.

**Review by kushal-arora-fw — APPROVED** [review]

File: src/frontend/src/pages/PPQFormPage.tsx (hunk 3 above)
Comment (binggaofw) [reply src/frontend/src/pages/PPQFormPage.tsx:271]: will do it in the integration ticket

File: src/frontend/src/pages/PPQFormPage.tsx (hunk 4 above)
Comment (binggaofw) [reply src/frontend/src/pages/PPQFormPage.tsx:241]: sure

**Pull request merged**